attributes/_configure.rb

```diff
@@ -42,7 +42,7 @@
     case node['platform_version'].to_i
     when 6
       '-systemd-managed 0'
-    when 7
+    when 7, 8
```

Comment:
I'm still not exactly sure what changed to require this. A few weeks ago, the vagrant boxes came up fine on centos 8.x with initd, and some of my tests to switch to systemd were actually failing (though I don't recall the exact error).  Now, trying to use initd fails and it works fine with systemd...